src/array/data_chunk_builder.rs

```diff
@@ -57,6 +57,31 @@ impl DataChunkBuilder {
         }
     }
 
+    /// Push a row of str in the Iterator.
+    ///
+    /// The row is accepted as an iterator of &str, and it's required that the size of row
+    /// should be the same as the number of columns.
+    ///
+    /// A [`DataChunk`] will be returned while `size == capacity`, and it should always be handled
+    /// correctly.
```

Comment:
```suggestion
    /// Push a row of `str` in the Iterator.
    ///
    /// The row is accepted as an iterator of `&str`, and it's required that the size of row
    /// should be the same as the number of columns.
    ///
    /// A [`DataChunk`] will be returned while `size == capacity`, and it should always be handled
    /// correctly.
    #[must_use]
```

Reply:
`Result` has already been marked as `#[must_use]`, so I guess there is no need to add one.